src/main/java/net/masterthought/cucumber/json/Step.java

```diff
@@ -154,11 +154,34 @@ public String getDocStringOrNothing() {
                 getDocString().getEscapedValue() +
                 "</div></div>";
     }
+    
+    private static int labelCnt = 0;
 
     private String formatError(String errorMessage) {
         String result = errorMessage;
         if (errorMessage != null && !errorMessage.isEmpty()) {
-            result = errorMessage.replaceAll("\\\\n", "<br/>");
+            if (errorMessage.startsWith( "<" )) {
+                result = errorMessage.replaceAll( "\\\\n", "<br/>" );
+            } else {
+                StringBuilder sb = new StringBuilder();
+                int id = ++labelCnt;
+                String[] s = errorMessage.split( "\\n" );
+                sb.append( "<input class=\"collapse\" id=\"_"+ id +"\" type=\"checkbox\"><label for=\"_" + id + "\">" );
+                //sb.append( StringEscapeUtils.escapeHtml( s[0] ) );
+                sb.append( s[0] );
+                sb.append( "</label><div>" );
+                
+                for ( int i = 1; i < s.length; i++) {
```

Comment:
apache commons should be able to join this loop in single instruction - take a look and find proper method